src/components/MemberContribution/ContributionAccordianItem.tsx

```diff
@@ -1,23 +1,25 @@
 import { RootState } from '@/src/store';
-import { AccordionPanel, Box, Text } from '@chakra-ui/react';
+import { Box, Text, Button, Link } from '@chakra-ui/react';
 import { useState } from 'react';
 import { useSelector } from 'react-redux';
 import DeliveryDetails from './DeliveryDetails';
 import NextLink from 'next/link';
-import { Button, Link } from '@chakra-ui/react';
 import styles from './memberContribution.module.css';
 import Image from 'next/image';
 import { useGetIsSuperUser } from '../../utils/customHooks';
+import {ContributionAccordionItemProps} from "./types"
+
+
+const URL =
+  process.env.NEXT_PUBLIC_TASK_BASE_URL ??
+  'https://status.realdevsquad.com/tasks/';
```

Comment:
- is the default value required here? instead of hardcoding the value if we can ensure the value is set when creating a build does it solve the issue?

Reply:
Yes right, this is not needed here. We can add it in prod env.